chia/util/condition_tools.py

```diff
@@ -86,35 +125,31 @@ def pkm_pairs(conditions: SpendBundleConditions, additional_data: bytes) -> Tupl
     for spend in conditions.spends:
```

Comment:
Here you might consider more meta-programming vs boilerplate with something like:

```python
condition_spend_pairs = [(ConditionOpcode.AGG_SIG_PARENT, spend.agg_sig_parent), (ConditionOpcode.AGG_SIG_PUZZLE, spend.agg_sig_puzzle), ...]
for condition, spends in condition_spend_pairs:
    for pk, msg in spends:
        ret[0].append(bytes48(pk))
        ret[1].append(make_aggsig_final_message(condition, msg, spend, data))
```

It would really be better if we could go `spend[condition]` so we wouldn't need pairs, but whatevs.

The names are pretty uniform, so I suppose we could go

```python
condition_spend_pairs = [
    (getattr(ConditionOpcode, condition_name.upper()), getattr(spend, condition_name)
    for condition_name in "agg_sig_parent agg_sig_puzzle ...".split())
   
```

Reply:
Thanks, done.